DuckDuckGo/PrivacyProtectionController.swift

```diff
@@ -54,10 +42,8 @@ class PrivacyProtectionController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        transitioningDelegate = self
-        popoverPresentationController?.backgroundColor = view.backgroundColor
-
-        initOmniBar()
+        navigationBar.isHidden = isPad
+        popoverPresentationController?.backgroundColor = UIColor(white: 240/255, alpha: 1.0)
```

Comment:
Where did that color came from? :) Shouldn't it be either `nearlyWhite` or `lightGreyish`?